packages/muelu/doc/TutorialReadTheDocs/pages/error_messages.rst

```diff
@@ -0,0 +1,195 @@
+==============
+Error Messages
+==============
+
+This section summarizes typical error messages and hints how to address the underlying problem.
+
+Syntax errors
+=============
+
+Parser errors
+--------------------------
+
+::
+
+    XML parse error at line 27: file ended before closing element 'ParameterList' from line 1
+
+It seems like you forgot to close the **<ParameterList>** section that is opened in line 1 of the xml file.
+Close it by adding the line **/ParameterList>** at the end of the (sub-)list.
```

Comment:
missing '<'

Reply:
Ah, I guess that's an intentional error

Reply:
No, that's actually an accidental error. Will fix it.